hyp3_autorift/process.py

```diff
@@ -65,29 +91,22 @@ def process(reference, secondary, download=False, polarization='hh', orbits=None
 
     product_dir = os.path.join(os.getcwd(), 'PRODUCT')
 
-    if not reference.is_file() or not secondary.is_file() and download:
-        log.info('Downloading Sentinel-1 image pair')
-        dl_file_list = 'download_list.csv'
-        with open('download_list.csv', 'w') as f:
-            f.write(f'{reference.name}\n'
-                    f'{secondary.name}\n')
-
-        execute(f'get_asf.py {dl_file_list}')
-        os.rmdir('download')  # Really, get_asf.py should do this...
-
-    if orbits is None:
-        orbits = Path('Orbits').resolve()
-        mkdir_p(orbits)
-        reference_state_vec, reference_provider = downloadSentinelOrbitFile(reference.stem, directory=orbits)
-        log.info(f'Downloaded orbit file {reference_state_vec} from {reference_provider}')
-        secondary_state_vec, secondary_provider = downloadSentinelOrbitFile(secondary.stem, directory=orbits)
-        log.info(f'Downloaded orbit file {secondary_state_vec} from {secondary_provider}')
-
-    if aux is None:
-        aux = orbits
+    if download:
+        for scene in [reference, secondary]:
+            if not scene.is_file():
+                log.info('Downloading Sentinel-1 image pair')
```

Comment:
this message is a little out of place now, could put it right after `if download:`, or drop it because `download_file` already logs `downloading <url>`